nix/cells/automation/pipelines.nix

```diff
@@ -0,0 +1,73 @@
+{ cell
+, inputs
+}:
+let
+  inherit (inputs.nixpkgs) system;
+
+  common =
+    { config
+    , ...
+    }: {
+      preset = {
+        nix.enable = true;
+
+        github-ci = {
+          # Tullia tasks can run locally or on Cicero.
+          # When no facts are present we know that we are running locally and vice versa.
+          # When running locally, the current directory is already bind-mounted
+          # into the container, so we don't need to fetch the source from GitHub
+          # and we don't want to report a GitHub status.
+          enable = config.actionRun.facts != { };
+          repo = "input-output-hk/plutus";
+        };
+      };
+
+      memory = 1024 * 8;
+      nomad.resources.cpu = 10000;
+    };
+in
+{
+  ci = { config, lib, ... }: {
+    imports = [ common ];
+
+    preset.github-ci.sha =
+      config.preset.github-ci.lib.readRevision inputs.cells.cloud.library.actions.ci.input null;
+
+    command.text = ''
+      nix build -L .#${lib.escapeShellArg system}.automation.ciJobs.required
+    '';
+  };
+
+  publish-unraveling-recursion-paper = { config, pkgs, lib, ... }: {
+    imports = [ common ];
+
+    preset.github-ci.sha =
+      config.preset.github-ci.lib.readRevision inputs.cells.cloud.library.actions.paper.input null;
+
+    command.text = ''
+      set -x
+
+      nix build -L .#${lib.escapeShellArg system}.plutus.packages.unraveling-recursion-paper
+
+      jq > value.json --null-input \
+        --arg revision "$(${lib.escapeShellArg config.preset.github-ci.sha})" \
+        '{"plutus/unraveling-recursion-paper": $revision}'
+
+      http --multipart --ignore-stdin "$CICERO_WEB_URL/api/run/$NOMAD_JOB_ID/fact" \
+        value@value.json binary@result/unraveling-recursion.pdf
+    '';
+
+    dependencies = with pkgs; [ jq httpie ];
```

Comment:
Why not just splice these into `command.text`?

Reply:
To have a normal-looking script instead of `${lib.getExe pkgs.jq}`, personal preference I guess.

Reply:
It's already got quite a lot of splicing garbage in there, but as you like :)